_plugins/download.rb

```diff
@@ -8,7 +8,7 @@ def initialize(tag_name, text, tokens)
 
     def render(context)
       version = context['site']['trino_version']
-      base = 'https://repo.maven.apache.org/maven2/io/prestosql'
```

Comment:
the apache URL is the official one for central btw..

Reply:
Not according to https://central.sonatype.org/pages/consumers.html

It doesn’t actually matter since they point to the same CDN run by Sonatype. This makes it consistent with our documentation links. Either way, we could trivially change it later if needed.

Reply:
Hahahah .. I wrote that page ages ago and it does not seem to have been updated. Anyway ... yes .. both DNS entries point to the CDN so they work the same.